actors/common/src/main/java/org/sunbird/bean/ClaimStatus.java

```diff
@@ -0,0 +1,17 @@
+package org.sunbird.bean;
+
+public enum ClaimStatus {
```

Comment:
Java doc , required for all class and public method.

Reply:
done